src/core/AnimatedEvent.js

```diff
@@ -58,9 +58,9 @@ function sanitizeArgMapping(argMapping) {
     };
 
     const proxy =
-      Platform.OS === 'android'
-        ? createEventObjectProxyPolyfill()
-        : new Proxy({}, proxyHandler);
+      typeof Proxy === 'function'
```

Comment:
```suggestion
      Proxy
```

@christianbach / cc @kmagiera 
How about this change? `Proxy` on Android is `undefined` I suppose

Reply:
Thats correct Proxy is undefined on "Android" with the JSC shipping with react-native and on iOS below 10

Reply:
nice, so I believe that my suggestion could be applied. It makes it more readable I suppose

Reply:
👍 I will implement both your suggestion and @kmagiera and update the PR. Thanks!

Reply:
Done, updated the PR with the suggested changes and added a bonus (linted and formated the code base)